Integrations/integration-CrowdStrikeFalcon.yml

```diff
@@ -0,0 +1,1095 @@
+        """
+        if severity in ('Critical', 'High'):
+            return 3
+        elif severity in ('Medium', 'Low'):
+            return 2
+        return 0
+
+
+    def get_trasnformed_dict(old_dict, transformation_dict):
+        """
+            Returns a dictionary with the same values as old_dict, with the correlating key:value in transformation_dict
+
+            :type old_dict: ``dict``
+            :param old_dict: Old dictionary to pull values from
+
+            :type transformation_dict: ``dict``
+            :param transformation_dict: Transformation dictionary that contains oldkeys:newkeys
+
+            :return Transformed dictionart (according to transformation_dict values)
+            :rtype ``dict``
+        """
+        new_dict = {}
+        for k in list(old_dict.keys()):
+            if k in transformation_dict:
+                new_dict[transformation_dict[k]] = old_dict[k]
+        return new_dict
+
+
+    def extract_transformed_dict_with_split(old_dict, transformation_dict_arr):
+        """
+            Extracts new values out of old_dict using a json structure of:
+            {'Path': 'Path to item', 'NewKey': 'Value of output key', 'Delim': 'Delimiter char', 'Index': Split Array Index}
+        """
+        new_dict = {}
+        for trans_dict in transformation_dict_arr:
+            try:
+                val = demisto.get(old_dict, trans_dict['Path'])
+                if 'split' in dir(val):
+                    i = trans_dict['Index']
+                    new_dict[trans_dict['NewKey']] = val.split(trans_dict['Delim'])[i]
+            except:
+                LOG('Error with: {}'.format(trans_dict))
+        return new_dict
+
+
+    ''' COMMAND SPECIFIC FUNCTIONS '''
+
+
+    def get_token():
+        """
+            Retrieves the token from the server if it's expired and updates the global HEADERS to include it
+
+            :rtype: ``str``
+            :return: Token
+        """
+        now = datetime.now()
+        ctx = demisto.getIntegrationContext()
+        if ctx:
+            time_delta = now - datetime.fromtimestamp(ctx.get('time'))
+            passed_mins = time_delta.seconds / 60
+            auth_token = get_token_request() if passed_mins >= TOKEN_LIFE_TIME else ctx.get('auth_token')
+        else:
+            auth_token = get_token_request()
+            demisto.setIntegrationContext({'auth_token': auth_token, 'time': date_to_timestamp(now) / 1000})
+        HEADERS['Authorization'] = 'Bearer {}'.format(auth_token)
+        return auth_token
+
+
+    def get_token_request():
+        """
+            Sends token request
+
+            :rtype ``str``
+            :return: Access token
+        """
+        body = {
+            'client_id': USERNAME,
+            'client_secret': PASSWORD
+        }
+        headers = {
+            'Authorization': HEADERS['Authorization']
+        }
+        token_res = http_request('POST', '/oauth2/token', data=body, headers=headers, safe=True)
```

Comment:
we need to check if there is any meaning for insecure mode in this integration. it is clearly disabled in the get token part. and maybe it should never be enabled.

Reply:
this is actually an argument I added, which means
 `If set to true will return None in case of http error`.

I can change the name to `err_flag` or something of that like